.github/workflows/ci.yaml

```diff
@@ -38,19 +38,48 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-
-      - name: Set up Flutter
-        uses: subosito/flutter-action@v2
-        with:
-          channel: stable
-
+      
       - uses: actions/setup-node@v4
       
       - name: npm install and build
         run: |
           npm install
           npm run build
 
+      - name: Bring in rubin_chart repo
+        uses: actions/checkout@v4
+        with:
+          repository: 'lsst-sitcom/rubin_chart'
+          path: rubin_chart
+          ref: 'deploy-slac'
+
+      - name: Bring in DDV Flutter app repo
+        uses: actions/checkout@v4
+        with: 
+          repository: 'lsst-ts/rubintv_visualization'
+          path: ddv
+          ref: 'deploy-slac'
+
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+
+      - name: Set up .env file
+        run: echo -e "ADDRESS=rubintv/ws/ddv" > ${{ github.workspace }}/ddv/.env
+
+      - name: Install dependencies
+        run: flutter pub get
+        working-directory: ./ddv
+
+      - name: Print flutter/dart version
+        run: flutter --version
+
+      - name: Build Flutter web app
+        run: flutter build web --base-href /rubintv/ddv/ --release
+        working-directory: ./ddv
```

Comment:
We need to make this `--base-href` configurable so we can deploy the rubintv app using other prefixes, such as the Summit-dev deployment which uses the `rubintv-dev` prefix. @ugyballoons please add a TODO here referencing a DM ticket to track the work :pray:.

Reply:
A different DM ticket should be created and referenced here too in regards to the branch configuration for the repos cloning, i.e. not using hard-coded branches.

Reply:
I've added the same ticket ref. to the `.env ADDRESS` var as the `--base-href` issue as they both need to be configured at runtime.